src/constants.js

```diff
@@ -0,0 +1,5 @@
+export const API_KEY = '8cac6dec66e09ab439c081b251304443'
```

Comment:
Consider enhancing our code security. The current constant.js file exposes an API key directly in the code, which can be risky if shared accidentally.

One way to implement this would be by updating the code in constant.js to fetch the API key from environment variables